xtas/tasks/_sentiwords/__init__.py

```diff
@@ -10,20 +10,18 @@
 
 #@worker_process_init.connect
 def load():
-    max_len = 0           # max n-gram length
+    max_len = 0           # max n-gram length minus one
     sentiment = {}
-    with open(_SENTI_PATH) as f:
-        for ln in f:
-            if ln.startswith('#'):
+    with open(_SENTI_PATH) as sentiwords_file:
+        for line in sentiwords_file:
+            if line.startswith('#'):
                 continue
 
-            w, prior = ln.rsplit('\t', 1)
+            word, prior = line.split('\t')
             prior = float(prior)
-            if prior == 0:
-                continue
```

Comment:
I suppose this is one of the superfluous checks. LGTM. I should probably have checked in a script to process new versions, but since it was a oneliner I didn't bother.